ACM Digital Library.js

```diff
@@ -56,7 +56,7 @@ function doWeb(doc, url) {
 }
 
 function getSearchResults(doc, checkOnly) {
-	var results = ZU.xpath(doc, '//tr/td/a[@target="_self"]'),
+	var results = ZU.xpath(doc, '//div[@id="results"]//a[@target="_self"]'),
```

Comment:
I would include `div[@class="title"]` in there as well. `@target="_self"` is not very unique.